packages/blade/src/components/Card/CardRoot.web.tsx

```diff
@@ -59,7 +63,7 @@ const CardRoot = ({
       as={as}
       {...props}
       isMobile={isMobile}
-      isPressed={isPressed}
+      isPressed={props.shouldScaleOnHover ? isPressed : false}
```

Comment:
also fixed another bug where on mobile we were changing pressed state even when shouldScaleOnHover is not set